app/models/postsModel.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Application\Models\PostsModel;
+
+//use Application\Lib\Database\DatabaseConnection;
+use Application\Core\Database\Database\DatabaseConnection;
+
+class PostsModel
```

Comment:
same le nom du file est PostModel au singulier aussi le nom de model par convention

Reply:
as ahead i remove the s and renaming the first class in class "PostNamingBase", everything working well after those changing